[DOC] Fix documentation issues in ray-observability directory (#56069)
ray-project/ray: doc/source/ray-observability/key-concepts.rst

Profiling
---------
Profiling is a way of analyzing the performance of an application by sampling the resource usage of it. Ray supports various profiling tools:

- CPU profiling for Driver and Worker processes, including integration with :ref:`py-spy <profiling-pyspy>` and :ref:`cProfile <profiling-cprofile>`
- Memory profiling for Driver and Worker processes with :ref:`memray <profiling-memray>`
- GPU profiling with :ref:`Pytorch Profiler <profiling-pytorch-profiler>` and :ref:`Nsight System <profiling-nsight-profiler>`
- Built in Task and Actor profiling tool called :ref:`Ray Timeline <profiling-timeline>`

> View :ref:`Profiling <profiling>` for more details. Note that this list is not comprehensive; feel free to contribute to it if you find other useful tools.

Reviewer: ```suggestion
View :ref:`Profiling <profiling>` for more details. Note that this list isn't comprehensive; feel free to contribute to it if you find other useful tools.
```

This to should be updated to include contribution instructions.
